sr_logging_common/logging/bag_rotate.py

```diff
@@ -40,29 +28,20 @@ def __init__(self, path, desired_bag_number, prefix):
         self.run()
 
     def remove_old_bags(self):
-        bags_to_remove = self.get_suffixed_bags(".bag")
-        bags_to_remove.extend(self.get_suffixed_bags(".bag.orig.active"))
+        bags_to_remove = self.get_file_paths_by_key(".bag")
         bags_to_remove.sort()
 
         for i, bag in enumerate(bags_to_remove):
             if i < len(bags_to_remove)-self._desired_bag_number - 1:
```

Comment:
Should we remove the `-1` ?
Like this i think we will be left with `self._desired_bag_number + 1` instead of the expected `self._desired_bag_number`.

Reply:
Perhaps it was done like this to count also on the bag that is currently active (the one that is recording data at the moment)

Reply:
Perhaps this -1 helps counting on the bag that is currently active. As in if self._desired_bag_number == 0 and len(existing_bags)==1; no bag would be removed